libs/portals/my-pages/core/src/components/SortableTable/useSortableData.ts

```diff
@@ -1,6 +1,36 @@
 import { useMemo, useState } from 'react'
 import { ConfigType } from './types'
 
+// Check if the date string is of the format dd.mm.yyyy
+// and return an object with day, month, and year
+const decunstructDateString = (dateString: string) => {
```

Comment:
```suggestion
const deconstructDateString = (dateString: string) => {
```

Reply:
No because the date is structured for the UI when rendering, this is added so that the table can also sort date strings